test/runtests.jl

```diff
@@ -4,17 +4,17 @@ using Compat.Test
 using Compat.Sockets
 import Compat.Libdl
 using Compat.Distributed
+using Compat: @info
 using Base: BufferStream
-using TestSetExtensions
 
-eval(Syslogs, Meta.parse("UDP_PORT = 8080"))
-eval(Syslogs, Meta.parse("TCP_PORT = 8080"))
+Core.eval(Syslogs, Meta.parse("UDP_PORT = 8080"))
```

Comment:
Can these just be formatted like `@eval Syslogs UDP_PORT = 8080` instead?

Reply:
Yep